### PR TITLE
add driver for 7.5in B HD display

### PR DESCRIPTION
**Pull request by ascension**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -79,6 +81,55 @@ const waveshare7in2v2Vertical = {
         this.driver.init();
     },
 };
+const renderCanvas = () => {
```
Comment (samsonmking) [inline devices.js:84]: @ascension can you provide a little more information about what this method is for? It seems like it's just writing the text 'Awesome!'?

File: devices.js [hunk 2]
```diff
@@ -79,6 +81,55 @@ const waveshare7in2v2Vertical = {
...
+    ctx.stroke();
+
+    return canvas.toBuffer();
+};
+
+const waveshare7in5bHDHorizontal = {
+    height: 528,
+    width: 880,
+    driver: waveshare7in5bHDDriver,
+    displayPNG: async function (imgContents, redImgContents) {
```
Comment (samsonmking) [inline devices.js:108]: @ascension for the other displays, this method is called in `page.js:28` where a PNG buffer browser screenshot is passed in as a single parameter. How was your workflow with these two parameters `imgContents` and `redImgContents`?

**Review by samsonmking — COMMENTED** [review]
@ascension Thanks for submitting this PR! I just have a couple comments (see above) about how your changes integrate with the overall usage of epaper.js

**Comment by ascension** [comment]
@samsonmking didn't intend to create the PR. I will open one I get a few of the kinks worked out.

**Pull request closed without merging**